.github/workflows/test-package.yml

```
@@ -23,7 +23,7 @@ jobs:
         sdk: [dev]
     steps:
       - uses: actions/checkout@v2
-      - uses: dart-lang/setup-dart@v0.1
+      - uses: dart-lang/setup-dart@v1.0
         with:
           channel: ${{ matrix.sdk }}
```

change `channel` to `sdk`
```suggestion
          sdk: ${{ matrix.sdk }}
```